manage/roles/agent-permissions.rst

```diff
@@ -17,10 +17,24 @@ Agent Permissions
 
    ...and give users access to new sidebar tabs for communicating with customers.
 
-:``chat.agent``:            :doc:`💬 Customer Chat </channels/chat>`
-:``cti.agent``:             :doc:`📞 Phone </system/integrations/generic-cti>` integration
-:``knowledge_base.editor``: :doc:`📕 Knowledge Base </manage/knowledge-base>` (create/edit privileges)
-:``knowledge_base.reader``: :doc:`📕 Knowledge Base </manage/knowledge-base>` (read privileges for internal & public content)
+:``chat.agent``:            `💬 Customer Chat <https://user-docs.zammad.org/en/latest/extras/chat.html>`_
+
+                            .. hint:: 🤓 Requires configuration of :doc:`Chat Channel </channels/chat>`
+:``cti.agent``:             Provides access to `📞 Caller Log <https://user-docs.zammad.org/en/latest/extras/caller-log.html>`_
+
+                            .. hint:: 🤓 Requires configuration of either integrations
+
+                               .. toctree::
+                                  :maxdepth: 1
+
+                                  Generic CTI </system/integrations/generic-cti>
+                                  placetel </system/integrations/placetel-cti>
+                                  sipgate </system/integrations/sipgate>
```

Comment:
Making this a `toctree` leads to these headings appearing in the sidebar TOC:

![Screenshot_20201002_230756](https://user-images.githubusercontent.com/12194123/94939055-327d1b80-0504-11eb-96c7-538b6ca51e49.png)

Was that on purpose? IMO it doesn't fit the organizational structure of the document.

Reply:
Good point. From my end it was on purpose because I technically like to see where it leads.
However: I do get your point that the structure virusally goes into the wrong direction. I'll revert this.

Reply:
will be address with commit 229e5a8c55fcf709e0eadb232d165c1959a47a63